### DISCUSSION TITLE
Layout Deployment Issues

### DISCUSSION THREAD

**Discussion by josodigital** [selldone/storefront#6] [Q&A]
I have done a french install and I am trying to deploy a new layout but I keep getting the error message "error_msg: 'You can not create more than 1 layouts! To have more layouts, please visit https://selldone.com/developer/layouts .' " See below.
 Yet when I visit https://selldone.com/developer/layouts and try to add new layouts, there are no options to do so. 

Any ideas how to resolve this?

Many thanks

Fleman

 INFO  Check out deployment instructions at https://cli.vuejs.org/guide/deployment.html

Done in 105.33s.

✅  Vue project built successfully.
▶  Creating ZIP file of the build output...
✅  Zip file created (12642906 total bytes).
▶  Uploading ZIP file...
Error: {
  error: true,
  code: 0,
  error_msg: 'You can not create more than 1 layouts! To have more layouts, please visit https://selldone.com/developer/layouts .'
}
❌  Upload error: TypeError: body used already for: https://api.selldone.com/developer/layouts/deploy

**Comment by pajuhaan** [comment]
We have set a limit for the number of layouts that can be created, and we've **increased it to 2**. Now you can add a new layout by uploading your layout.
![image](https://github.com/selldone/storefront/assets/4279061/690a2b77-0a6d-4e6e-aa67-a317e4091616)
![image](https://github.com/selldone/storefront/assets/4279061/681c18a9-a030-49e6-b005-9c38c5275eda)

Also you have **unlimited times** to upload **versions** for a layout (just keep the **"package"** name the same in the **manifest.json**). This limit on layouts per user will be removed in the future.
